libvirt/tests/src/virsh_cmd/domain/virsh_save.py

```
@@ -71,6 +71,7 @@ def run(test, params, env):
 
     if progress:
         options += " --verbose"
+    virsh.domstate(vm_name, ignore_status=True, debug=True)
```

Here ignore_status=True ,do you intend to allow silent failure in some situations ?

---

Yes, I don't want this cmd failure to result in failing on virsh.save case.